README.md

```diff
@@ -31,24 +31,37 @@
 
 ### Hooks
 
+Available hooks and a short summary of their purpose:
+
+#### State tools
+
 - `useArray` - Simplify array state management with useful utilities
-- `useDeepCompareEffect` - Run function when dependencies change, checked deeply for equality
 - `useFlag` - Simply manage boolean state
-- `useFlexCorners` - Dynamically find corners in items wrapped with flex
+- `useTally` - Increment and decrement numeric state
+
+#### Listeners/Observers
+
 - `useMutationObserver` - Observe a Node and rerender on mutations + get mutation information
 - `useResizeObserver` - Observe an Element and rerender on resizes + get size information
-- `useTally` - Increment and decrement numeric state
 - `useWindowListener` - Easily add and remove listeners on the window
 
+#### Styling tools
+
+- `useFlexCorners` - Dynamically find corners in items wrapped with flex
+
+#### Utilities
+
+- `useDeepCompareEffect` - Run function when dependencies change, checked deeply for equality
+- `useThrottleFunction` - Run a function when its args change up to a maximum number of calls per time period in ms
+- `useThrottleValue` - Reduce frequency of state updates on a variable to a maximum number of updates per time period in ms
+
 ## Contributing
 
-+ Pick a ticket from the GitHub project's issues and drop a comment to have it assigned to you. 
-+ Make sure branch names are descriptive and commit messages are valuable and in the imperative mood. 
-+ When a change is ready, simply open a PR and make sure to link it to the issue it references.
+See the [contribution guide](.github/contributing.md) for details on getting started and contributing to the project.
 
-<br>
+All are welcome to open issues and PRs in the styles described in the contributing guide.
 
->Note that all PRs will be squashed, so PR titles must also adhere to the commit message rules. Ensure they are descriptive, concise, and in the imperative mood. For example: "Add useFlag hook" rather than "New hook" or "Added useFlag hook". The title will be the commit message that remains in the `main` history, it is important they are well-written.
+When opening PRs, make sure to use the [pull request template](.github/pull_request_template.md)
 
```

Comment:
These points might just be redundant since the contributing guide we're pointing them to in the previous sentence has the info needed.